doc/classes/FileAccess.xml

```diff
@@ -417,14 +417,14 @@
 				Stores an integer as 8 bits in the file. This advances the file cursor by 1 byte.
 				[b]Note:[/b] The [param value] should lie in the interval [code][0, 255][/code]. Any other value will overflow and wrap around.
 				[b]Note:[/b] If an error occurs, the resulting value of the file position indicator is indeterminate.
-				To store a signed integer, use [method store_64], or convert it manually (see [method store_16] for an example).
+				To store a signed integer, use [method store_64], or convert it manually (see [method store_16] for an example). Returns [code]true[/code] if the operation is successful.
```

Comment:
This additional sentence should be moved to the first line. It does not make sense here.